doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,106 @@
+.. _ref_make_container:
+
+Create your Fluent Docker container
+===================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps in this section.
+
+The following procure for creating a Fluent Docker container uses a local
+Ubuntu machine to copy the needed files from the Fluent
+
+
+Requirements
```

Comment:
_Prerequisites_ is a more appropriate term because it means what's required beforehand or necessary as preparation for something else.

Reply:
Done. Thank you.